plugins/woocommerce-blocks/assets/js/blocks/product-collection/edit/index.tsx

```diff
@@ -24,9 +31,33 @@ const Edit = ( props: ProductCollectionEditComponentProps ) => {
 		[ clientId ]
 	);
 
-	const Component = hasInnerBlocks
-		? ProductCollectionContent
-		: ProductCollectionPlaceholder;
+	const productCollectionUIStateInEditor =
+		getProductCollectionUIStateInEditor( {
+			location,
+			attributes: props.attributes,
+			hasInnerBlocks,
+			usesReference: props.usesReference,
+		} );
```

Comment:
Super nit, but that changing order increases readability 😅 
```suggestion
		getProductCollectionUIStateInEditor( {
			location,
			hasInnerBlocks,
			attributes: props.attributes,
			usesReference: props.usesReference,
		} );
```

Reply:
Thanks for being nitpicky. I will make the change. 🙌🏻